packages/config/README.md

```diff
@@ -2,6 +2,74 @@
 
 ## Usage
 
+### `.bemrc` fields
+
+Field   | Type    | Purpose
+---     | ---     | ---
+root    | `Boolean` | Used to determine the root directory. Configs in parent dirs won't be gathered
+naming  | `string`, `Object` | Name of one of [naming presets](https://github.com/bem/bem-sdk/tree/master/packages/naming.presets) or custom naming definition
+levels  | `Array<LevelConf>` | List of known levels in the right order<br> (usually local) with their configurations
+sets    | `Object<string, SetConf>` | Named sets of layers to build
+libs    | `Object<string, LibraryConf>` | Dependency libraries to use in sets
+plugins | `Object<string, PluginConf>` | Various configurations for plugins,<br>can be reached via [`.module`](#module) method
+
+#### `root`
+
+For the following file structure
+```
+/
+    projects/
+        prj1/
+            .bemrc
+        .bemrc
+    .bemrc
+```
+and `/projects/prj1/` as current working directory `root` option set to `true` in `/projects/prj1/.bemrc` will prevent `bem-config` to collect data from `/projects/.bemrc` and `/.bemrc`.
+
+#### `LevelConf`
+
+Describes [redefinition level](https://en.bem.info/methodology/key-concepts/#redefinition-level) with sources — a set of layers relative to library path (`.bemrc` location)
+and depends on naming preset. E.g. `common` and `desktop` for `bem-components/` (library) path and [`origin`](https://github.com/bem/bem-sdk/blob/master/packages/naming.presets/origin.js) preset.
```

Comment:
"a set of layers relative to library path (...)" - is very good.

The second part is not.  Can we simply add some examples:

`{ layer: 'common', path: 'foo/bar' }` defines set of one layer with relative path `'foo/bar'`
`{ layer: 'common' }` defines set of layers depending on naming. Relative paths can be `'common.blocks/'`, `'blocks-common'`, etc.

Am I right?

Reply:
You are right.

We want to merge `levels` into `libraries` (as it is semantically the same) but @tadatuta worries about classic definition of `level`. 
What you think if we just move this part to the `library` description with any custom fields?